src/profiler.lua

```diff
@@ -106,6 +125,20 @@ local function ops_pooling(module, input)
     return batch_size * output_width * output_height * kernel_ops
 end
 
+local function ops_unpooling(module, input)
+    assert(input:dim() == 4, "ops_unpooling supports only batched inputs!")
+    local batch_size = input:size(1)
+    local input_planes = input:size(2)
+    local input_height = input:size(3)
+    local input_width = input:size(4)
+
+
+    local output_width = input_width
```

Comment:
So you are calculating only theoretical ops and not based implementation.
I have modified it now.